### ISSUE TITLE
Miscellaneous QA Tasks for V3.3.0 

### ISSUE THREAD

**Issue by WillHaack** [ByunLab/staRt-ionic#491]
There are a few pushes I made without taking the time to do handle the overhead of making issues on github. This issue is simply a QA task that handles a few new items in v3.3.0

1. Redone Login Flow

- [x] Log out of the app. Make sure that the login screen is centered horizontally and in the upper ~75% of the screen with a nice backdrop of a lighthouse.

- [x] Log back into an account that already has a list of participants. Make sure that the participant list loads without having to click around on the app.

- [x] Log out of the app. Close the app with the home button. Open the app and repeat the above step.

- [x] Repeat the above 3 steps, except this time when you log back in, log into newly created accounts. Make sure that everything looks correct on the initial profiles page.

2. Formal Session Navigation (Fixing the problem with navigation on the last step)

- [x] Start a formal session sequence, and continue it to completion. Make sure that on the last step you are navigated to the final quiz.

3. Prompt "Would you like to upload" before navigating away from a quest or quiz.

- [x] Start a quest. Give two ratings. Then navigate away by clicking on a different tab on the left sidebar. Make sure you are prompted ~ "Would you like to upload the file?" BEFORE the page switches to the new tab you clicked. (You can click "later" on the prompt)

- [x] Repeat the above step, but this time start a quiz.

4. Fixed bug where quiz started with "0/30" or "0/50" and the first word is not displayed until the rating button is clicked. 

- [x] Start a quiz. Make sure every time you start the quiz the first word displays correctly. Do this 15 times. Each time you start a quiz change up the actions you did before starting the quiz, i.e. navigate from each different tab, switch participants in the profiles page first, play around and try to recreate the bug.

5. Button clicks cause two actions (i.e. a rating gives two ratings.)

- [ ] We know these bugs exist. There is a similar bug where when you navigate away sometimes you are prompted "Would you like to stay?" twice in a row. Try to find a way to reproduce these bugs and describe the steps to reproduce them. My experience has been by quickly navigating around + giving ratings very quickly, you can sometimes get yourself into a bugged state.

**Comment by yl5504** [comment]
2. Formal Session Navigation (Fixing the problem with navigation on the last step)
Start a formal session sequence, and continue it to completion. Make sure that on the last step you are navigated to the final quiz.
I was navigated to free play, then quest, then after I complete quest, here is the pop-up screen, it did not navigate me to quiz.
![IMG_0026](https://user-images.githubusercontent.com/43120402/70661613-a88d5a80-1c32-11ea-93e4-97dd284beff2.PNG)

@helencarey

**Comment by helencarey** [comment]
Thanks @yl5504!
I just moved the 'Formal Session Navigation' issue back to its own ticket for @WillHaack.

If QA #5 is OK, then you can close this ticket.
